shuup/xtheme/locale/fi_FI/LC_MESSAGES/django.po

```diff
@@ -84,6 +84,30 @@ msgstr "Tyylit"
 msgid "The fonts, colors, and styles to use with your theme."
 msgstr "Teemasi käyttämät fontit, värit ja tyylit."
 
+msgid "Content in this placeholder is shown without limitations."
+msgstr "Tämä sisältöalue näytetään ilman rajoituksia."
+
+#, python-format
+msgid ""
+"Content in this placeholder is shown for %(category_name)s category only."
+msgstr "Tämä sisältöalue näytetään vain kategorialle %(category_name)s."
```

Comment:
Maybe: `Tämän sisältöalueen sisältö näytetään vain kategorialle %(category_names)s.` (reason being it mentions "content" on English string and it's missing here) 

Also, see lines 95-109 as well below if you decide to fix this as they have the same "problem".

Reply:
Untouched since we couldn't agree. Feel free to make update this at https://www.transifex.com/shuup/shuup/translate/#fi_FI I am happy to add you to the project.